crates/hc_deepkey_sdk/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "hc_deepkey_sdk"
+version = "0.6.0"
+authors = ["Matthew Brisebois <matthew.brisebois@holo.host>"]
+edition = "2018"
+license = "CAL-1.0"
+repository = "https://github.com/holochain/deepkey"
+description = "SDK for the Deepkey DNA Zomes"
+readme = "README.md"
+
+[lib]
+name = "hc_deepkey_sdk"
```

Comment:
I changed this from `deepkey_sdk` to `hc_deepkey_sdk`

Reply:
Is this lib section necessary? Can't we use defaults if it's the same name and the default lib output types?

Reply:
Most of our other crates have it